src/ignitus-AboutPage/ignitus-CoreTeam/Components/index.test.tsx

```diff
@@ -3,33 +3,30 @@ import { render } from 'enzyme';
 import { CoreTeam } from './index';
 // import { Data } from '../constants';
 
-React.useLayoutEffect = React.useEffect;
-
 it('should be rendered', () => {
   const tree = render(<CoreTeam />);
   expect(tree).toMatchSnapshot();
 });
 
-// describe('<CoreTeam />', () => {
-// let wrapper;
-// beforeEach(() => {
-//   wrapper = mount(<CoreTeam />);
-// });
+describe('<CoreTeam />', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = render(<CoreTeam />);
+  });
 
-// it('should render correct title', () => {
-//   expect(wrapper.find('h2').text()).toBe(' Our Team ');
-// });
+  it('should render correct title', () => {
+    expect(wrapper.find('h2').text()).toBe(' Our Team ');
+  });
 
-// it('should render correct core team data', () => {
-//   console.log('Team: ', wrapper.find('TeamItem').length);
-//   expect(wrapper.find('TeamItem').length).toBe(Data.length);
-//   Data.forEach((val: any, index: number) => {
-//     expect(
-//       wrapper
-//         .find('TeamItem')
-//         .at(index)
-//         .props().item === val,
-//     );
-//   });
-// });
-// });
+  // it('should render correct core team data', () => {
+  //   expect(wrapper.find('TestItem').length).toBe(Data.length);
+  //   Data.forEach((val: any, index: number) => {
+  //     expect(
+  //       wrapper
+  //         .find('img')
+  //         .at(index)
+  //         .props().item === val,
+  //     );
+  //   });
+  // });
```

Comment:
@Marvin9 Can you pls check out the test cases here whenever you get the chance? I am not sure why the test fails here; though I have tried some other alternatives as well, but neither of them seems to work :/

Reply:
Root cause is ```IntersectionObserver``` is treated as undefined or something that's why not component but errorBoundry is rendering. [Mock IntersectionObserver beforeEach test case](https://stackoverflow.com/a/62198850).